perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/widgets/items/Password.java

```diff
@@ -43,6 +45,8 @@ protected Widget initWidget() {
 
 		widget = new InputGroup();
 
+		GWT.log("Init Widget");
```

Comment:
not sure what is this log for?

Reply:
Just a leftover from testing. Nice catch!